tests/test_cli/test_cli.py

```diff
@@ -33,8 +38,8 @@ def test_derive_schema(self):
         result = self.runner.invoke(main, cmd)
         self.assertEqual(0, result.exit_code)
         out = result.stdout
-        schema = yaml_loader.loads(str(out), SchemaDefinition)
-        expected = yaml_loader.load(str(PERSONINFO_TGT_SCHEMA), SchemaDefinition)
+        # schema = yaml_loader.loads(str(out), SchemaDefinition)
+        # expected = yaml_loader.load(str(PERSONINFO_TGT_SCHEMA), SchemaDefinition)
```

Comment:
variables are unused but it looks like you were going to do something with them... possibly?